polaris-react/src/components/AlphaStack/AlphaStack.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import type {spacing} from '@shopify/polaris-tokens';
+
+import {classNames, variationName} from '../../utilities/css';
+import {elementChildren} from '../../utilities/components';
+
+import styles from './Stack.scss';
+
+type SpacingTokenGroup = typeof spacing;
+type SpacingTokenName = keyof SpacingTokenGroup;
+
+// TODO: Bring this logic into tokens
+type Spacing = SpacingTokenName extends `space-${infer Scale}` ? Scale : never;
+
+type Align = 'start' | 'end' | 'center';
+
+export interface AlphaStackProps {
+  /** Elements to display inside stack */
+  children?: React.ReactNode;
+  /** Adjust spacing between elements */
+  spacing?: Spacing;
+  /** Adjust vertical alignment of elements */
+  align?: Align;
+}
+
+export const AlphaStack = ({children, spacing, align}: AlphaStackProps) => {
```

Comment:
```suggestion
export const AlphaStack = ({children, spacing, align = 'start'}: AlphaStackProps) => {
```

Then we wouldn't need to add [align-items to the root class](https://github.com/Shopify/polaris/pull/7036/files#diff-5e37364e8a0f34fdc64878bc60adbd92eb5a565d3206740794e47a98b7e4f4b7R5) in the css file

Reply:
I think we should also be opinionated around providing a default spacing for `gap`

Reply:
Just out of curiosity, in what situation would we need the `gap` default value to be different from the default stack spacing value?

Reply:
They're one and the same in my mind. Let me know if you want to jump into a huddle and jam for a bit